css/index.css

```diff
@@ -1,3 +1,346 @@
-/* Example of @import use
-This imports the `reset.css` file */
 @import './reset.css';
+
```

Comment:
Let the `index.css` file include only imports of other css file.
You can divide your CSS into different media queries. See example below:
```css
/* Your reset css file at the top so that other style can overwrite it */

@import './reset.css';

/* Flex box classes (This is just an example, you can ommit thi if your flex box styles 
or equivalent is in the media query section */
@import './flex.css';

/* This will be the main styles where anything else apart from the media quries are */
@import './main.css';

/* Then the follwoi ng are sets of media queries import, with this the files gets 
imported only when the condition is satified, this helps you maintan yur conde even 
better as you know where to look/change your code for a specific screen size style */
@import 'screens/small-screen.css' screen and (min-width: 576px);
@import 'screens/small-screen-only.css' screen and (max-width: 780px);
@import 'screens/medium-screen.css' screen and (min-width: 780px);
@import 'screens/large-screen.css' screen and (min-width: 1020px);
```

The above is only an example, you can fine-tune to our tase, however, leave this file for only imports.

Reply:
Thanks for the comment. I'll make the necessary adjustments.